import-service/controllers/parse-s3-csv.controller.ts

```diff
@@ -1,13 +1,14 @@
 import { HttpStatusCode } from './../../product/utils/http-status-codes';
 import { AppError } from './../utils/app-error.utils';
 import { S3EventRecord } from 'aws-lambda';
-import { S3 } from 'aws-sdk';
+import { S3, SQS } from 'aws-sdk';
 import csvParser from 'csv-parser';
 
 export const parseCSVFiles = async (
   records: S3EventRecord[]
 ): Promise<void> => {
   const s3 = new S3();
```

Comment:
A suggestion to create all clients outside the lambdas. That way you will save resources and money avoiding long execution times.